ClipboardAPI/tag-security-privacy-clipboard-svg.md

```diff
@@ -0,0 +1,97 @@
+### Questions from https://www.w3.org/TR/security-privacy-questionnaire/
+
+## 2. Questions to Consider
+
+### 2.1. What information might this feature expose to Web sites or other parties, and for what purposes is that exposure necessary?
+
+This feature exposes `image/svg+xml` MIME type to the clipboard using the async clipboard API to support copy-paste of SVG images. All the restrictions related to secure context, permission etc apply for reading this format. The format is parsed using the strict HTML processing that strip out security sensitive tags.
+
+### 2.2. Do features in your specification expose the minimum amount of information necessary to enable their intended uses?
+
+Yes.
+
+### 2.3. How do the features in your specification deal with personal information, personally-identifiable information (PII), or information derived from them?
+
+No PII information is exposed in this new clipboard format support.
+
+### 2.4. How do the features in your specification deal with sensitive information?
+
+This feature doesn't deal with any sensitive information.
+
+### 2.5. Do the features in your specification introduce new state for an origin that persists across browsing sessions?
+
+No.
+
+### 2.6. Do the features in your specification expose information about the underlying platform to origins?
+
+No.
+
+### 2.7. Does this specification allow an origin to send data to the underlying platform?
+
+No.
```

Comment:
Arguably the answer is "yes" here, since this is a way to put a new kind of data on the platform clipboard. But we could say that this is just an extension of an existing API for putting data on the platform clipboard, and the data will be sanitized before it's exposed to the platform.

Reply:
Done.